hack/toolchain_install.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
```

Comment:
Maybe add this?
```suggestion
set -e
set -o pipefail
```

Reply:
ATM it's not needed as no piped command is there. shellcheck `check-extra-masked-returns` should prevent us from adding a piped command without adding `-o pipefail` or splitting the command. So we can add this in future if needed